rendermodules/mesh_lens_correction/do_mesh_lens_correction.py

```diff
@@ -254,27 +233,49 @@ def run(self):
                     args=['--output_json', out_file.name])
             pmgen.run()
 
+        # load these up in memory to pass to actual solver
+        rawtilespecs = renderapi.tilespec.get_tile_specs_from_z(
+                ts_example['output_stack'],
+                ts_example['z'],
+                render=renderapi.connect(**ts_example['render']))
+        rawtdict = [t.to_dict() for t in rawtilespecs]
+        matches = renderapi.pointmatch.get_matches_within_group(
+                self.args['match_collection'],
+                self.args['sectionId'],
+                render=renderapi.connect(**self.args['render']))
+
         self.logger.setLevel(self.args['log_level'])
 
+        solver_args = {
+                'nvertex': self.args['nvertex'],
+                'regularization': self.args['regularization'],
+                'good_solve': self.args['good_solve'],
+                'tilespecs': rawtdict,
+                'matches': matches,
+                'output_dir': self.args['output_dir'],
+                'outfile': 'resolvedtiles.json.gz',
+                'compress_output': False,
+                'log_level': self.args['log_level']}
+
         meshclass = MeshAndSolveTransform(
-                input_data=args_for_input,
-                args=['--output_json', out_file.name])
+                input_data=solver_args, args=[])
         # find the lens correction, write out to new stack
         meshclass.run()
+        with open(meshclass.args['output_json'], 'r') as f:
```

Comment:
using an `output_json`  populated within the MeshAndSolveTransform module based on the `output_dir` ( I think?) will make this module less robust to name collisions in cases where the same lens correction is double-run (which used to happen fairly often in automated processing, but may not at this point.)

Reply:
I guess there are 2 things here:
the output_json could be overwritten, and, the resolved json.gz could be overwritten. If these were timestamped filenames, would that alleviate the concern?

Reply:
I'd be happy with that or being able to specify individual output files independent of the `output_dir`.

Reply:
v0.2.3 of `em_stitch` now has a timestamp boolean argument. I'm leaving the default as False, so as not to rock the microscope boat right now, but, hard-coding it to True here.
https://github.com/AllenInstitute/em_stitch/commit/c3439cf2a22703cd3c5c2daff620d53c05a334f8